app.js

```diff
@@ -0,0 +1,6 @@
+const config = require('./config/config.json');
+const { User, Product } = require('./models');
```

Comment:
point 4 from evaluation criteria isn't pass. You need to rewrite code with es6 import/export syntax. Avoid require